.circleci/config.yml

```diff
@@ -25,13 +25,32 @@ jobs:
           path: bundles/core/target/surefire-reports
       - run:
           name: Upload Code Coverage
-          command: bash <(curl -s https://codecov.io/bash)
+          command: bash <(curl -s https://codecov.io/bash) -F unittests
+
+  karma:
+    docker:
+      - image: circleci/node:lts-browsers
+    steps:
+      - checkout
+      - run:
+          name: Provision
+          command: npm install
+          working_directory: ./ui.apps
+      - run:
+          name: Run Unit Tests
+          command: npm test
+          working_directory: ./ui.apps
+      - store_test_results:
+          path: ui.apps/junit
```

Comment:
Shall we use another folder name? `junit` is for java.

Reply:
I configured karma to use junit as output format, so that CircleCI can parse it. To avoid confusion, I renamed the folder to `karma-junit`.